### PR TITLE
update ping probe

### PR DESCRIPTION
**Pull request by AnjaBruls**
## Description

First update to the ping probe.

### CODE REVIEW

File: lib/check/base.py [hunk 1]
```diff
@@ -1,21 +1,77 @@
...
 class Base:
     interval = 300
     required = False
+    type_name = None
 
     @classmethod
     async def run(cls, data, asset_config=None):
         try:
-            asset_id = data['hostUuid']
+            asset_id = data['hostUuid']  # TODO remove?
```
Comment (joente) [inline lib/check/base.py:19]: If not used, maybe comment the line. It still might be useful to quickly know how to get the asset_id

File: lib/check/base.py [hunk 2]
```diff
@@ -1,21 +1,77 @@
...
+        else:
+            return state_data
+
+    @classmethod
+    async def get_data(cls, address, count, interval, timeout):
+        data = None
+        try:
+            data = await cls.run_check(address, count, interval, timeout)
+        except Exception as err:
+            logging.exception(f'Ping error: `{err}`\n')
```
Comment (joente) [inline lib/check/base.py:51]: The address might be useful in the exception, is this provided by the `err` variable of do you need to add the `address` to the log line?

File: lib/check/base.py [hunk 3]
```diff
@@ -1,21 +1,77 @@
...
+            raise
+
+        try:
+            state = cls.get_result(data)
+        except Exception:
+            logging.exception('Ping parse error\n')
```
Comment (joente) [inline lib/check/base.py:57]: At least add the `address` as otherwise you have no clue which address was being pinged

**Review by joente — CHANGES_REQUESTED** [review]
I think that only the exceptions need to be improved

File: lib/check/base.py (hunk 1 above)
Comment (AnjaBruls) [reply lib/check/base.py]: see [623fb06](https://github.com/oversight/pingprobe/pull/2/commits/623fb0643592eb1e8f1626e000e922e5255b121a)

File: lib/check/base.py (hunk 2 above)
Comment (AnjaBruls) [reply lib/check/base.py]: I added the address to the log line. See [623fb06](https://github.com/oversight/pingprobe/pull/2/commits/623fb0643592eb1e8f1626e000e922e5255b121a)

File: lib/check/base.py (hunk 3 above)
Comment (AnjaBruls) [reply lib/check/base.py]: I added the address to the log line. See [623fb06](https://github.com/oversight/pingprobe/pull/2/commits/623fb0643592eb1e8f1626e000e922e5255b121a)

File: lib/check/base.py [hunk 4]
```diff
@@ -1,21 +1,77 @@
...
 
+from .utils import check_config
+
+DEFAULT_PING_COUNT = 5  # (1 - 9)
+DEFAULT_PING_INTERVAL = 1  # (1s - 9s)
+DEFAULT_PING_TIMEOUT = 10
+
 
 class Base:
-    interval = 300
```
Comment (joente) [inline lib/check/base.py:5]: This `interval` is not the same interval as parsed to the ping check. The "interval" for the ping check is the interval between each ping. This "interval" is the interval between each time to perform the check.

**Review by joente — APPROVED** [review]

**Pull request merged**